src/java.base/macosx/classes/sun/nio/ch/KQueuePort.java

```diff
@@ -40,17 +41,15 @@
  * AsynchronousChannelGroup implementation based on the BSD kqueue facility.
  */
 
-final class KQueuePort
-    extends Port
-{
+final class KQueuePort extends Port {
     // maximum number of events to poll at a time
     private static final int MAX_KEVENTS_TO_POLL = 512;
 
     // kqueue file descriptor
     private final int kqfd;
 
     // address of the poll array passed to kqueue_wait
-    private final long address;
+    private final MemorySegment pollArrayRegions;
```

Comment:
Can we drop "Regions" from the name? It's an array of kevent structures so pollArray will do.